markdown/nextflow_tutorial.md

```diff
@@ -17,7 +17,7 @@ process < name > {
   output:
    < process outputs >
 
-  [script|shell|exec]:
+  script:
```

Comment:
Whilst this is true, it's very rare that processes use anything other than `script` so I thought it better to simplify this.